e2e/tests/utils/test-data-manager.ts

```
@@ -0,0 +1,354 @@
+    switch (scenario) {
+      case 'empty':
+        await this.resetDatabase();
+        return {
+          users: [],
+          speeches: [],
+          timestamp: new Date().toISOString(),
+          testRun
+        };
+        
+      case 'basic':
+        const basicUser = await this.createUser();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)